vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -721,6 +737,36 @@ void BatGetMedia::onGetChannelIdFromUserPage(
   }
 }
 
+void BatGetMedia::onFetchDataFromNonEmbeddable(
+    uint64_t windowId,
+    const ledger::VisitData& visit_data,
+    const std::string& providerType,
+    const uint64_t& duration,
+    const std::string& media_key,
+    const std::string& mediaURL,
+    int response_status_code,
+    const std::string& response,
+    const std::map<std::string, std::string>& headers) {
+  if (response_status_code != 200) {
+    onMediaActivityError(visit_data, providerType, windowId);
+    return;
+  }
+
+  if (providerType == YOUTUBE_MEDIA_TYPE) {
+    std::string publisherName = parsePublisherName(response);
+    std::string publisher_url =
+        getPublisherUrl(parseChannelId(response), providerType);
+
+    ledger_->LoadURL(publisher_url,
```

Comment:
nit: stack params one after another so that is easier to read them, like https://github.com/brave/brave-core/pull/1756/files#diff-501b5889b2700e1f1a74c584464a569cR332